dashboard/src/scenes/territory/list.js

```diff
@@ -89,7 +89,13 @@ const List = () => {
             onSortBy: setSortBy,
             sortOrder,
             sortBy,
-            render: (territory) => <b>{territory.name}</b>,
+            render: (territory) => {
+              return (
+                <div style={{overflowWrap: 'anywhere'}}>
```

Comment:
dans l'idéal, même si je sais que là y'en a partout (c'est une dette dont on a hérité), on veut éviter les `style={{ }}` parce que
- `We recommend to only use the style attribute when your styles depend on JavaScript variables.` -> https://beta.reactjs.org/reference/react-dom/components/common#applying-css-styles
- pour être cohérent au maximum avec l'utilisation de tailwind (même si Tailwind ce sont des classes prédéfinies et que là c'est pas le cas...)
- parce que c'est moche d'avoir les deux, je trouve (ce n'est que mon avis)
- parce que pour beaucoup, ce n'est pas une bonne pratique d'avoir des objets "inline", c'est-à-dire définis à la volée, parce que ça peut affecter les performances (NA dans notre cas je pense, mais bon à avoir en tête quand même)


ce n'est qu'une suggestion, j'accepte qu'elle soit 🥊 bien sûr

```suggestion
                <div className="[overflow-wrap:anywhere]">
```

Reply:
on peut aussi modifier la config tailwind pour rajouter une classe plus propre aussi, si on veut - mais bon, moi je préfère ça